mlir/test/Conversion/GPUToROCDL/gpu-to-rocdl.mlir

```diff
@@ -740,3 +740,25 @@ gpu.module @test_module {
 gpu.module @test_custom_data_layout attributes {llvm.data_layout = "e"} {
 
 }
+
+// -----
+
+gpu.module @test_module {
```

Comment:
Can you ad a test that exercises the >=gfx10 path and the new pass option?

Reply:
added a test.

Reply:
I don't see a test that uses the `subgroup-size` pass optio